sudachipy/dictionarylib/dictionarybuilder.py

```diff
@@ -91,7 +91,7 @@ def build_lexicon(self, lexicon_input_stream):
         line_no = -1
         try:
             for i, row in enumerate(csv.reader(lexicon_input_stream)):
-                line_no = -1
+                line_no = i
```

Comment:
Thanks for your report!
I think the error log will not be displayed if the format of the first line is not valid.
The if statement at line 100 should be modified as follows: `if line_no > -1:` (or `if line_no >= 0`).
Could you fix this?

Reply:
Yes, I overlooked it and fixed it. Thank you for your review!